unlock-app/src/components/interface/Errors.tsx

```diff
@@ -4,18 +4,20 @@ import styled from 'styled-components'
 import { resetError } from '../../actions/error'
 import Buttons from './buttons/layout'
 import ErrorMessage from '../helpers/ErrorMessage'
-import { Error } from '../../unlockTypes' // eslint-disable-line no-unused-vars
+/* eslint-disable */
+import { UnlockError, WarningError, isWarningError } from '../../utils/Error'
+/* eslint-enable */
 
 interface Props {
-  errors: Error[]
-  close: (errorName: string) => any
+  errors: WarningError[]
```

Comment:
Warnings and fatal errors are now distinct types, and those types are handled by separate components